sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -69,6 +69,15 @@ object StaticSQLConf {
       .checkValue(cacheSize => cacheSize >= 0, "The maximum size of the cache must not be negative")
       .createWithDefault(1000)
 
+  val FILESOURCE_TABLE_CATALOG_CACHE_EXPIRE_SECONDS =
+    buildStaticConf("spark.sql.filesourceTableCatalogCacheExpireSeconds")
+      .internal()
+      .doc("The maximum expire seconds time of cache table catalog.")
+      .intConf
+      .checkValue(cacheSize => cacheSize >= 0 && cacheSize < 8,
```

Comment:
`cacheSize` seems copied from existing config items. Need to be rewording.

Reply:
OK